CONTRIBUTING.md

```diff
@@ -64,7 +73,7 @@ Exa)
 
 If you're adding a capability onto an already existing version requirement, simply add your object into the existing array.
```

Comment:
nit: "If you're adding a capability **to** an already existing version requirement, simple add your object **to** the existing array."

(Bold to emphasize which words should be edited)

Reply:
Sounds good, changes pushed!